testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/security/callerprincipal/GetCallerPrincipalTestCase.java

```diff
@@ -234,7 +235,8 @@ public void testStatefulLifecycle() throws Exception {
             return null;
         };
         try {
-            Util.switchIdentitySCF("user1", "password1", callable);
+            //Util.switchIdentitySCF("user1", "password1", callable);
+            callable.call();
```

Comment:
Unless I'm missing something the callable should just be removed and its code inlined.